<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LuversControl</class>
 <widget class="QWidget" name="LuversControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>57</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="schemaBox">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="margin">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">background: transparent</string>
          </property>
          <property name="text">
           <string>Схема</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="perimetrBox">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>периметр</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="margin">
         <number>2</number>
        </property>
        <item>
         <widget class="QFrame" name="subPerimetr">
          <property name="styleSheet">
           <string notr="true">QFrame {background: rgba(170, 0, 127, 0)}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="1" column="0">
              <widget class="QLabel" name="label_5">
               <property name="styleSheet">
                <string notr="true">background: transparent</string>
               </property>
               <property name="text">
                <string>штук</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_6">
               <property name="styleSheet">
                <string notr="true">background: transparent</string>
               </property>
               <property name="text">
                <string>шаг,мм</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1" rowspan="2">
              <widget class="LuversAmount" name="perimetrAmount" native="true"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="customFrame">
       <property name="styleSheet">
        <string notr="true">QFrame {background:transparent;}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QGroupBox" name="leftBox">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="title">
           <string>лево</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <property name="margin">
            <number>2</number>
           </property>
           <item>
            <widget class="LuversAmount" name="leftAmount" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="upBox">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="title">
           <string>вверх</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="margin">
            <number>2</number>
           </property>
           <item>
            <widget class="LuversAmount" name="upAmount" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="rightBox">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="title">
           <string>право</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="margin">
            <number>2</number>
           </property>
           <item>
            <widget class="LuversAmount" name="rightAmount" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="downBox">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="title">
           <string>низ</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="margin">
            <number>2</number>
           </property>
           <item>
            <widget class="LuversAmount" name="downAmount" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LuversAmount</class>
   <extends>QWidget</extends>
   <header>widgets/work/task/luvers/luversamount.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
